scan.go

```diff
@@ -1227,13 +1245,10 @@ func stringToCm(token string) (e, m uint8, ok bool) {
 		e = 0
 		val = cmeters
 	}
-	for val > 10 {
+	for val >= 10 {
 		e++
 		val /= 10
 	}
-	if e > 9 {
```

Comment:
this became redundant due to the check of `meters > 90000000 || (meters == 90000000 && cmeters != 0)`, but if you choose not to adopt that change this check will have to be kept.